xdsl/printer.py

```diff
@@ -261,12 +270,15 @@ def _print_new_line(
         self, indent: int | None = None, print_message: bool = True
     ) -> None:
         indent = self._indent if indent is None else indent
-        self.print_string_raw("\n")
+        print(file=self.stream)
```

Comment:
Can you add a comment to make it obvious?

Reply:
To make what obvious?

Reply:
I added a couple of comments, not sure if they're what you expected